src/Testcontainers.Firestore/FirestoreBuilder.cs

```diff
@@ -0,0 +1,74 @@
+namespace Testcontainers.Firestore;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class FirestoreBuilder : ContainerBuilder<FirestoreBuilder, FirestoreContainer, FirestoreConfiguration>
+{
+    const string Command = "gcloud beta emulators firestore start --host-port 0.0.0.0:8080";
```

Comment:
Please use the exec form in `Init()`, like:

    .WithEntrypoint("gcloud")
    .WithCommand("beta", "emulators", "firestore", "start", "--host-port", "0.0.0.0:" + FirestorePort)

Reply:
done